gitpod-io/gitpod
# feat(supervisor): add resource status severity

## Description
- Extends `status.proto` with ResourceStatus severity to remove the duplicated severity calculation from each client
- Refactor `gp-cli` and `jetbrains backend-plugin` to consume the new severity property

## Related Issue(s)

Fixes #12075

## How to test
1. Launch a workspace https://afalz-1207846d967e0b.preview.gitpod-dev.com/#https://github.com/gitpod-io/gitpod
2. Select IntelliJ latest as preferred IDE
3. Stress the CPU/Memory and using both `gp top` and JB backend control center, see the cpu/memory thresholds change color to WARN/DANGER

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
